demo-shell/src/app/components/files/files.component.html

```diff
@@ -25,16 +25,6 @@
                 </adf-toolbar-title>
 
                 <div class="app-document-action-buttons">
-                    <button
-                        data-automation-id="create-new-folder"
-                        mat-icon-button
-                        [disabled]="!canCreateContent(documentList.folderNode)"
```

Comment:
can you please also check if this function no longer used, and remove it as well if it's a dead code

Reply:
I returned this button because it is used in tests